modes/calendar/evil-collection-calendar.el

```diff
@@ -99,7 +134,11 @@
     ;; quit
     "q" 'calendar-exit
     "ZQ" 'evil-quit
-    "ZZ" 'calendar-exit))
+    "ZZ" 'calendar-exit)
+
+  (defvar evil-collection-setup-calendar-org-bindings)
```

Comment:
Maybe move this defvar at the top-level with a docstring?

Reply:
The `defvar` is used to "declare" `evil-collection-setup-calendar-org-bindings` which is defined in `evil-collection.el`.

`evil-collection-ivy.el` does the same.